indico/modules/rb_new/client/js/common/rooms/RoomEditModal.jsx

```diff
@@ -90,9 +64,147 @@ class RoomEditModal extends React.Component {
         onClose();
     };
 
+    handleSubmit = async (formData) => {
+        const {actions: {updateRoom}, room: {id}} = this.props;
+        let rv = await updateRoom(id, formData);
+        if(rv.error){
+            return rv.error;
+        }
+    };
+
+    renderHeader = (title) => (
+        <Header key={title}>
+            <Translate>{title}</Translate>
```

Comment:
`<Translate>` doesn't work like this since it needs to extract the strings for translation. you need to translate the literal string